backend/curriculum_tracking/api_views.py

```diff
@@ -883,6 +891,26 @@ def auto_assign_reviewers(self, request, pk=None):
             response = actor.send()
             return Response({"status": "OK", "data": response.asdict()})
 
+    @action(
+        detail=False,
+        methods=["POST"],
+        serializer_class=serializers.SetDueTimeSerializer,
+        permission_classes=[HasObjectPermission(permissions=Team.PERMISSION_MANAGE_CARDS)]
+    )
+    def bulk_set_due_dates(self, request, pk=None):
+        serializer = self.get_serializer(data=request.data)
+        if serializer.is_valid():
+            teams = self.get_object()   # This will get you all the teams, we only need the teams in the request data
+            for team in teams:
+                if team.name in request.data.get('team'):
+                    team_cards = get_team_cards(team, request.data.get('content_item'))
+
+            for card in team_cards:
+                AgileCard.set_due_time(card, request.data.get('due_time'))
+            return Response(f'Due dates set for these team cards: {[card for card in team_cards]}')
```

Comment:
The frontend expects a JSON response.  
Generally when we have an api endopoint that edits an instance we have the endpoint return that instance. That way the frontend can know what alterations were made just by looking at the response instead of having to implement any intelligence.

Eg. Take a look at the "start_project" action on the AglieCard viewset. 

It would be good to have a response that the frontend (or any client application) could find useful.  I'd recommend making use of the `CardSummarySerializer`